benchmark/galsim/galsim_field.py

```diff
@@ -11,20 +11,22 @@
 FITS_CATALOG_FILENAME = os.path.join('output', 'galsim_field_500_catalog.fits')
 ARCSEC_PER_DEGREE = 3600.
 FIELD_EXPAND_ARCSEC = 10
+SKY_LEVEL_NMGY = 0.155 # similar to SDSS
+COUNTS_PER_NMGY = 180.0 # similar to SDSS
 
 def set_image_dimensions(test_case, catalog_rows):
     min_ra_deg = min(float(row['right_ascension_deg']) for row in catalog_rows)
     max_ra_deg = max(float(row['right_ascension_deg']) for row in catalog_rows)
     min_dec_deg = min(float(row['declination_deg']) for row in catalog_rows)
     max_dec_deg = max(float(row['declination_deg']) for row in catalog_rows)
 
-    width_arcsec = (max_ra_deg - min_ra_deg) * ARCSEC_PER_DEGREE + 2 * FIELD_EXPAND_ARCSEC
-    height_arcsec = (max_dec_deg - min_dec_deg) * ARCSEC_PER_DEGREE + 2 * FIELD_EXPAND_ARCSEC
+    height_arcsec = (max_ra_deg - min_ra_deg) * ARCSEC_PER_DEGREE + 2 * FIELD_EXPAND_ARCSEC
+    width_arcsec = (max_dec_deg - min_dec_deg) * ARCSEC_PER_DEGREE + 2 * FIELD_EXPAND_ARCSEC
```

Comment:
Yeah I'm not sure what the best way is...but I did write a painstaking description of the situation, in case you didn't see it: https://github.com/jeff-regier/Celeste.jl/wiki/Coordinate-systems